### ISSUE TITLE
fix eclipse "output dir" config to be independent of ant build

### ISSUE THREAD

**Issue by dlaboss** [quarks-edge/quarks#86]
There are annoying interactions between the ant build tooling and eclipse (building/cleaning) resulting from an overlap in some locations of their javac output directories.  Going to change things so they're independent of each other.

Leaving the ant stuff as is: class files under "classes" directories - one or more per Java-Project.

Going to change the eclipse build configs to start using the default eclipse project convention for the output dir: <project>/bin.

**Comment by home4slc** [comment]
Thanks Dale - this has been an annoyance to me too. ;-)

**Comment by ddebrunner** [comment]
Any change of a pull request before/on Monday?

**Comment by dlaboss** [comment]
I created and merged pull #103 
I recommend the following after updating your workspace with this merge:

```
$ cd <quarks-edge>
$ ant clean
$ rm -rf `find . -name classes -d`  # any residual eclipse created classes dirs
restart eclipse  # may have to Project > Clean... all
```
